geom/src/polyline.rs

```diff
@@ -123,6 +123,12 @@ impl PolyLine {
         PolyLine::must_new(pts)
     }
 
```

Comment:
How about a docstring comment describing the output? I have to remember what `rem_euclid` does. Something like `/// Returns the quadrant where the overall angle of this polyline (pointing from the first to last point) is in. Output between 0 and 3.`